hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ReplicatedMapCodecTemplate.java

```diff
@@ -49,7 +49,7 @@
      * @param name Name of the ReplicatedMap
      * @return the number of key-value mappings in this map.
      */
-    @Request(id = 2, retryable = true, response = ResponseMessageConst.INTEGER)
+    @Request(id = 2, retryable = true, response = ResponseMessageConst.INTEGER, partitionIdentifier = "random")
     Object size(String name);
```

Comment:
I does not matter. It can choose random when and use that always or choose random for each request. We are for example using a random partition id per proxy.